index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <title>Tsige's Personal Portifolio</title>
+</head>
+
+<body>
+  <nav class="toolbar">
+    <!-- <div class="logo"><a href="">Tsige Wossne</a></div> -->
+    <div class="menu_main">
+      <input type="checkbox" class="toggler">
+      <div class="humbuger"></div>
+      <div class="menu">
+        <div class="overlay">
+          <div>
+            <ul>
+              <li><a href="">Home</a></li>
+              <li><a href="">Projects</a></li>
+              <li><a href="">Contribution</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+  </nav>
+  <section class="headline">
+    <div class="container">
+      <h1>Hey there. <br>
+        I’m Tsige.</h1>
+      <h1 class="green">I’m a software developer! </h1>
```

Comment:
[optional] line 38 should be in a `p` tag for SEO purposes and good semantics, you should have only one `h1` tag in your html file 😄